File: faculdade/A4-PLP/ConversorDeUnidades.py
def metros_para_centimetros(metros):
    return metros * 100

def metros_para_milimetros(metros):
    return metros * 1000

def metros_para_quilometros(metros):
    return metros / 1000

def metros_para_pes(metros):
    return metros * 3.28084

def metros_para_jardas(metros):
    return metros * 1.09361

def metros_para_milhas(metros):
    return metros / 1609.344

def exibir_conversoes(metros):
    print(f"\nConversões para {metros:.2f} metros:")
    print(f"Centímetros: {metros_para_centimetros(metros):.2f} cm")
    print(f"Milímetros: {metros_para_milimetros(metros):.2f} mm")
    print(f"Quilômetros: {metros_para_quilometros(metros):.5f} km")
    print(f"Pés: {metros_para_pes(metros):.2f} ft")
    print(f"Jardas: {metros_para_jardas(metros):.2f} yd")
    print(f"Milhas: {metros_para_milhas(metros):.5f} mi")

def main():
    try:
        metros = float(input("Digite a medida em metros: "))
        exibir_conversoes(metros)
    except ValueError:
        print("Por favor, insira um número válido!")

File: faculdade/A4-PLP/ConversorDeUnidades.java
import java.util.Scanner;

public class ConversorDeUnidades {

    private double metros;

    public ConversorDeUnidades(double metros) {
        this.metros = metros;
    }

    public double getMetros() {
        return metros;
    }

    public void setMetros(double metros) {
        this.metros = metros;
    }

    public double metrosParaCentimetros() {
        return metros * 100;
    }

    public double metrosParaMilimetros() {
        return metros * 1000;
    }

    public double metrosParaKilometros() {
        return metros / 1000;
    }

    public double metrosParaPes() {
        return metros * 3.28084;
    }

    public double metrosParaJardas() {
        return metros * 1.09361;
    }

    public double metrosParaMilhas() {
        return metros / 1609.344;
    }

    public void exibirConversoes() {
        System.out.println("\nConversões para " + metros + " metros:");
        System.out.println("Centímetros: " + metrosParaCentimetros() + " cm");
        System.out.println("Milímetros: " + metrosParaMilimetros() + " mm");
        System.out.println("Quilômetros: " + metrosParaKilometros() + " km");
        System.out.println("Pés: " + metrosParaPes() + " ft");
        System.out.println("Jardas: " + metrosParaJardas() + " yd");
        System.out.println("Milhas: " + metrosParaMilhas() + " mi");
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite a medida em metros: ");
        double metros = scanner.nextDouble();

        ConversorDeUnidades conversor = new ConversorDeUnidades(metros);

        conversor.exibirConversoes();

        scanner.close();
    }
}